chapter_notation/index.md

```diff
@@ -1,79 +1,94 @@
+* $\sum_i=1^n$: summation over collection with indices ranging from $1$ to $n$
+* $\prod$: product over collection with indices ranging from $1$ to $n$
+* $\stackrel{\mathrm{def}}{=}$: an equality asserted as a definition of the symbol on the left-hand side
 
 
 ## Calculus
 
-* $\frac{dy}{dx}$: Derivative of $y$ with respect to $x$
-* $\frac{\partial y}{\partial x}$: Partial derivative of $y$ with respect to $x$
-* $\nabla_{\mathbf{x}} y$: Gradient of $y$ with respect to $\mathbf{x}$
-* $\int_a^b f(x) \;dx$: Definite integral of $f$ from $a$ to $b$ with respect to $x$
-* $\int f(x) \;dx$: Indefinite integral of $f$ with respect to $x$
+* $\frac{dy}{dx}$: derivative of $y$ with respect to $x$
+* $\frac{\partial y}{\partial x}$: partial derivative of $y$ with respect to $x$
+* $\nabla_{\mathbf{x}} y$: gradient of $y$ with respect to $\mathbf{x}$
+* $\int_a^b f(x) \;dx$: definite integral of $f$ from $a$ to $b$ with respect to $x$
+* $\int f(x) \;dx$: indefinite integral of $f$ with respect to $x$
 
 ## Probability and Information Theory
 
-* $P(\cdot)$: Probability distribution
-* $Z \sim P$: Random variable $Z$ has probability distribution $P$
-* $P(X \mid Y)$: Conditional probability of $X \mid Y$
-* $p(x)$: Probability density function
-* ${E}_{x} [f(x)]$: Expectation of $f$ with respect to $x$
-* $X \perp Y$: Random variables $X$ and $Y$ are independent
-* $X \perp Y \mid Z$: Random variables  $X$  and  $Y$  are conditionally independent given random variable $Z$
-* $\mathrm{Var}(X)$: Variance of random variable $X$
-* $\sigma_X$: Standard deviation of random variable $X$
-* $\mathrm{Cov}(X, Y)$: Covariance of random variables $X$ and $Y$
-* $\rho(X, Y)$: Correlation of random variables $X$ and $Y$
+* $X$: a random variable
+* $P$: a probability distribution
+* $X \sim P$: the random variable $X$ has distribution $P$
+* $P(X=x)$: the probability assigned to the event where random variable $X$ takes value $x$
+* $P(X \mid Y)$: the conditional probability distribution of $X$ given $Y$
+* $p(\cdot)$: a probability density function (PDF) associated with distribution P
+* ${E}[X]$: expectation of a random variable $X$
+* $X \perp Y$: random variables $X$ and $Y$ are independent
+* $X \perp Y \mid Z$: random variables  $X$  and  $Y$ are conditionally independent given $Z$
+* $\sigma_X$: standard deviation of random variable $X$
+* $\mathrm{Var}(X)$: variance of random variable $X$ qual to $\sigma^2_X$
+* $\mathrm{Cov}(X, Y)$: covariance of random variables $X$ and $Y$
+* $\rho(X, Y)$: the Pearson correlation coefficient between $X$ and $Y$, equals $\frac{\mathrm{Cov}(X, Y)}{\sigma_X \sigma_Y}$
 * $H(X)$: Entropy of random variable $X$
-* $D_{\mathrm{KL}}(P\|Q)$: KL-divergence of distributions $P$ and $Q$
+* $D_{\mathrm{KL}}(P\|Q)$: the KL-divergence (or relative entropy) from distribution $Q$ to distribution $P$---measures the cost incurred for coding samples from $P$ with a scheme optimized for $Q$.
```

Comment:
KL-divergence is defined and explained in the main text: no need to define it in the notation